roxctl/collector/supportpackages/upload/command.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/stackrox/rox/roxctl/common/flags"
 )
 
+const (
+	uploadHelpLong = `Upload files from a collector support package to Central.
```

Comment:
nit: Dunno if it is intentional, but I've noticed the difference in capitalization between (here) "**c**ollector support package to **C**entral" and (below, l35 new) "Upload files from a **C**ollector support package to **C**entral".